docs/codeql/codeql-language-guides/analyzing-data-flow-in-csharp.rst

```diff
@@ -439,7 +439,7 @@ Exercise 2
      Configuration() { this="String to System.Uri" }
      
      override predicate isSource(DataFlow::Node src) {
-       src.asExpr().hasValue()
+       src.asExpr().(Expr).hasValue()
```

Comment:
This change is not needed, as https://github.com/github/codeql/issues/4848 will fix it instead.

Reply:
OK. Thanks for your reply.

Reply:
@hvitved removed.